components/blocks/Actions.tsx

```diff
@@ -101,11 +122,12 @@ export const Actions = ({ items, align = 'left' }) => {
 
 export const CodeButton = ({ children, ...props }) => {
   const [copied, setCopied] = React.useState(false)
+  const id = sanitizeLabel(children.toString())
```

Comment:
Instead of using the children which can be anything - instead can we pass down some unique value (if thats whats intended)

Reply:
made it children[0] (the label of the object)